File: task/task-common/src/main/scala/com/atguigu/userprofile/bean/TagInfo.scala
package com.atguigu.userprofile.bean

import java.util.Date
import java.lang.Long
case class TagInfo(var id:Long,
                   var tagCode:String,
                   var tagName:String,
                   var tagLevel:Long,
                   var parentTagId:Long,
                   var tagType:String,
                   var tagValueType:String,
                   var  tagValueLimit:Long,
                   var  tagValueStep:Long,
                   var  tagTaskId:Long,
                   var  tagComment:String,
                   var  createTime:Date,
                   var  updateTime:Date
                  ) {
  def this()  ={
    this(null,null,null,null,null,null,null,null,null,null,null,null,null )
  }
}

File: manager/src/main/java/com/atguigu/userprofile/bean/TagInfo.java
package com.atguigu.userprofile.bean;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.Date;
import java.io.Serializable;

/**
 * <p>
 * 
 * </p>
 *
 * @author zhangchen
 * @since 2021-04-13
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class TagInfo implements Serializable {

    private static final long serialVersionUID = 1L;
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    private String tagCode;

    private String tagName;

    private Long tagLevel;

    private Long parentTagId;

    @TableField(exist = false)
    private Long parentTagLevel;

    @TableField(exist = false)
    private String parentTagCode;

    @TableField(exist = false)
    private String parentTagName;

    private String tagValueType;


    private String tagType;

    private Long tagTaskId;

    private String tagComment;

    private Date createTime;
    @TableField(exist = false)
    private String taskStatus;


}
